lib/ProxySQL_Admin.cpp

```diff
@@ -259,35 +292,43 @@ bool admin_handler_command_proxysql(char *query_no_space, unsigned int query_no_
 			proxy_debug(PROXY_DEBUG_ADMIN, 4, "Starting ProxySQL following PROXYSQL START command\n");
 			SPA->send_MySQL_OK(&sess->client_myds->myprot, NULL);
 		} else {
-			proxy_debug(PROXY_DEBUG_ADMIN, 4, "ProxySQL was already started when received PROXYSQL START command\n");
+			proxy_warning("ProxySQL was already started when received PROXYSQL START command\n");
 			SPA->send_MySQL_ERR(&sess->client_myds->myprot, (char *)"ProxySQL already started");
 		}
 		return false;
 	}
 
 	if (query_no_space_length==strlen("PROXYSQL RESTART") && !strncasecmp("PROXYSQL RESTART",query_no_space, query_no_space_length)) {
-		proxy_debug(PROXY_DEBUG_ADMIN, 4, "Received PROXYSQL RESTART command\n");
+		proxy_info("Received PROXYSQL RESTART command\n");
 		__sync_bool_compare_and_swap(&glovars.shutdown,0,1);
 		glovars.reload=1;
 		return false;
 	}
 
 	if (query_no_space_length==strlen("PROXYSQL STOP") && !strncasecmp("PROXYSQL STOP",query_no_space, query_no_space_length)) {
-		proxy_debug(PROXY_DEBUG_ADMIN, 4, "Received PROXYSQL STOP command\n");
+		proxy_info("Received PROXYSQL STOP command\n");
 		__sync_bool_compare_and_swap(&glovars.shutdown,0,1);
 		glovars.reload=2;
 		return false;
 	}
 
 	if (query_no_space_length==strlen("PROXYSQL SHUTDOWN") && !strncasecmp("PROXYSQL SHUTDOWN",query_no_space, query_no_space_length)) {
-		proxy_debug(PROXY_DEBUG_ADMIN, 4, "Received PROXYSQL SHUTDOWN command\n");
+		proxy_info("Received PROXYSQL SHUTDOWN command\n");
 		__sync_bool_compare_and_swap(&glovars.shutdown,0,1);
 		glovars.reload=0;
 		return false;
 	}
 
+	if (query_no_space_length==strlen("PROXYSQL FLUSH LOGS") && !strncasecmp("PROXYSQL FLUSH LOGS",query_no_space, query_no_space_length)) {
```

Comment:
Log rotation.
FLUSH LOGS opens and closes the files.
If a user want to create a new file, moves/delete/rename the old file and run FLUSH LOGS